ui/app/AppLayouts/Communities/panels/OverviewSettingsPanel.qml

```diff
@@ -133,9 +134,11 @@ StackLayout {
             topPadding: 0
             loginType: root.loginType
             communityName: root.name
-            //TODO connect to backend
-            isControlNode: root.owned
-            onPrimaryButtonClicked: isControlNode = !isControlNode
+            isControlNode: root.isControlNode
+            onPrimaryButtonClicked: {
+                if(isControlNode)
```

Comment:
May we log a warning in else?

Reply:
The primary button has 2 states: Import control node; Export control node. This is why a warning won't help. The else is just a state that's not implemented yet.

But I'll need to update this and make the footer API clear on this. I think it will be better to replace `primaryButtonClicked` with `importControlNodeClicked` and `exportControlNodeClicked` based on footer state.